packages/terra-outline-table-view/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "terra-outline-table-view",
+  "private": true,
+  "main": "lib/index.js",
+  "version": "0.1.0",
+  "description": "The terra-outline-table-view component ",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git",
+    "directory": "packages/terra-outline-table-view"
+  },
+  "keywords": [
+    "Cerner",
+    "Outline",
+    "Terra",
+    "terra-outline-table-view",
+    "Table",
+    "UI",
+    "View"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "peerDependencies": {
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5"
```

Comment:
Readme.md file has one more peer dependency `react-intl`. it is missing here :)

Reply:
added here https://github.com/cerner/terra-core/pull/3651/commits/639048e63b7776165b0ce16a2af9d6ce67405738#diff-5f92e8897649e36af2fd07d1abafe46a89d61e8f79446285b4a4d69ef2a260ffR30